app/src/main/java/com/cornellappdev/android/eatery/MenuActivity.java

```diff
@@ -298,15 +360,31 @@ public static Bitmap decodeSampledBitmapFromResource(
     return BitmapFactory.decodeResource(res, resId, options);
   }
 
-  /** Gets name of corresponding picture to cafe* */
+  /**
+   * Gets name of corresponding picture to cafe*
+   */
   public static String convertName(String str) {
-    if (str.equals("104West!.jpg")) return "104-West.jpg";
-    if (str.equals("McCormick's.jpg")) return "mccormicks.jpg";
-    if (str.equals("Franny's.jpg")) return "frannys.jpg";
-    if (str.equals("Ice Cream Cart.jpg")) return "icecreamcart.jpg";
-    if (str.equals("Risley Dining Room.jpg")) return "Risley-Dining.jpg";
-    if (str.equals("Martha's Express.jpg")) return "Marthas-Cafe.jpg";
-    if (str.equals("Bus Stop Bagels.jpg")) return "Bug-Stop-Bagels.jpg";
+    if (str.equals("104West!.jpg")) {
+      return "104-West.jpg";
+    }
+    if (str.equals("McCormick's.jpg")) {
+      return "mccormicks.jpg";
+    }
+    if (str.equals("Franny's.jpg")) {
+      return "frannys.jpg";
+    }
+    if (str.equals("Ice Cream Cart.jpg")) {
+      return "icecreamcart.jpg";
+    }
+    if (str.equals("Risley Dining Room.jpg")) {
+      return "Risley-Dining.jpg";
+    }
+    if (str.equals("Martha's Express.jpg")) {
+      return "Marthas-Cafe.jpg";
+    }
+    if (str.equals("Bus Stop Bagels.jpg")) {
+      return "Bug-Stop-Bagels.jpg";
+    }
```

Comment:
This is the exact same function as before, probably should be refactored into a `static` method instead!

Reply:
Will make a note of it, I disagree with the implementation of the function in general so maybe in the next merge we can clean it up.